prover-sever/src/server_main.rs

```diff
@@ -52,6 +52,9 @@ impl Rpc for RpcImpl {
     /// # Returns
     /// ProofResult instance which includes proof and final pair.
     fn prove(&self, trace: String, proof_type: i32) -> Result<ProofResult> {
+        #[cfg(feature = "mock-server")]
+        return Ok(ProofResult::new(vec![0; 128], Some(vec![0; 4640])));
+
```

Comment:
Please introduce a `MockPpcImpl` that implements `Rpc` traits.

Reply:
@dongchangYoo Please check this

Reply:
I have added an explanation of the mock-server for this function!